packages/core/src/App/Containers/Layout/header/traders-hub-header.tsx

```diff
@@ -99,61 +104,37 @@ const TradersHubHeader = observer(() => {
                         <DerivBrandLogo className='traders-hub-header__logo' />
                     </StaticUrl>
                 </div>
-                <DesktopWrapper>
-                    <div className='traders-hub-header__divider' />
-                    <TradersHubHomeButton />
-                </DesktopWrapper>
+                {is_desktop && (
+                    <React.Fragment>
+                        <div className='traders-hub-header__divider' />
+                        <TradersHubHomeButton />
+                    </React.Fragment>
+                )}
                 <MenuLinks {...{ is_traders_hub_routes: true }} />
```

Comment:
```suggestion
                <MenuLinks is_traders_hub_routes />
```

not related but can we just replace this line as well. You can ignore this comment 🙏

Reply:
this change does not work until MenuLinks is migrated to TS.